minorminer/utils/parallel_embeddings.py

```diff
@@ -333,8 +449,52 @@ def find_sublattice_embeddings(
             "source graphs must a graph constructed by "
             "dwave_networkx as chimera, pegasus or zephyr type"
         )
+    if tile_embedding is not None:
+        if not _is_valid_embedding(tile_embedding, S, tile, one_to_iterable):
+            raise ValueError("tile_embedding is invalid for S and tile")
+        else:
+            use_filter = False  # Unnecessary
+
+    if use_tile_embedding is None:
+        use_tile_embedding = tile == S  # Trivial 1:1
+        if use_tile_embedding and tile_embedding is None:
+            tile_embedding = {i: i for i in tile.nodes}
+            use_filter = False  # Unnecessary
+
+    if use_filter or (use_tile_embedding and tile_embedding is None):
+        # Check defect-free tile embedding is viable
+
+        # With respect to filter:
+        # * This assumes that an embedder that returns on a graph G
+        # will also return an embedding for any graph where G is
+        # a subgraph. Can be violated given heuristic search
+        defect_free_embs = find_multiple_embeddings(
+            S,
+            tile,
+            max_num_emb=1,
+            use_filter=False,
+            seed=seed,
+            embedder=embedder,
+            embedder_kwargs=embedder_kwargs,
+            timeout=timeout_at - perf_counter(),
```

Comment:
Could this cause issues if timeout is too small (but still >0) and thus `timeout_at - perf_counter()` becomes <= 0?

Perhaps worth adding a `if timeout_at - perf_counter() <= 1` directly above or changing this argument to `timeout=max(timeout_at - perf_counter(), 1)`, or something relevant?

Reply:
Timeouts smaller than 1 are allowed for these functions. A separate timeout can be given as part of embedder_kwargs where there could be problems?